src/System.Collections/src/System.Collections.csproj

```diff
@@ -40,7 +42,7 @@
       <TargetGroup>uap101aot</TargetGroup>
     </ProjectReference>
   </ItemGroup>
-  <ItemGroup Condition="'$(TargetGroup)' == 'netcoreapp' or '$(TargetGroup)' == 'uap101aot'">
+  <ItemGroup Condition="'$(TargetGroup)' == 'netcoreapp' or '$(TargetGroup)' == 'uap' or '$(TargetGroup)' == 'uap101aot'">
```

Comment:
>uap101aot [](start = 110, length = 9)

uap101aot we already shipped, right? do we plan to rip all mention of it out throughout, now?